tests/functional/spec/household-relationship.spec.js

```diff
@@ -44,4 +44,26 @@ describe('Household relationship', function() {
     expect(SummaryPage.isOpen()).to.be.true
   })
 
+  it('Given I answer the relationship questions for Joe Bloggs when I click previous on the Jane Doe relationship question then the relationships for John Bloggs should still be selected', function() {
+    // Given
+    startQuestionnaire(schema)
+    HouseholdCompositionPage.setPersonName(0, 'Joe Bloggs')
+      .addPerson()
+      .setPersonName(1, 'Jane Doe')
+      .addPerson()
+      .setPersonName(2, 'John Doe')
+      .submit()
+    HouseholdRelationshipPage.clickWhoIsRelatedHusbandOrWife(0)
```

Comment:
Minor point, but a lot of these methods start with `click`, perhaps they should be renamed to `select` now that the implementation has changed from radio field to a select dropdown.